package.json

```diff
@@ -90,6 +91,7 @@
     "fastest-validator": "^0.6.15",
     "glob": "^7.1.3",
     "ipaddr.js": "^1.9.0",
+    "lock": "^1.1.0",
```

Comment:
Please try to avoid to use a new dependency. I've checked the `lock` code and very simple. It stores the keys in an `Object`. However, it is not a Promise-based lib, so it generates a lot of code to convert it to promises. Would be good to create an own simple lock class, which store keys in a `Map` and all methods are Promise based. Could you make it?

Reply:
OK, I will remove this lib.